internal/services/servicebus/servicebus_namespace_disaster_recovery_config_data_source.go

```diff
@@ -37,25 +39,6 @@ func dataSourceServiceBusNamespaceDisasterRecoveryConfig() *pluginsdk.Resource {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
```

Comment:
should `namespace_id` be required? then made optional in the `features` if block?

Reply:
Oops! Commented on the wrong question:

You can make it Required but you have to set it as Optional in the FivePointOhBeta block so it's not a breaking change until 5.0 hits.

Reply:
I don't think this has been resolved yet? This should be marked as `Required` but changed to `Optional` in the `FivePointOh` block below